aten/src/ATen/native/TensorShape.cpp

```diff
@@ -774,10 +774,12 @@ Tensor cat(TensorList tensors, Dimname dim) {
 
 // torch.concat, alias for torch.cat
 Tensor& concat_out(TensorList tensors, Dimname dim, Tensor& result) {
+  TORCH_CHECK(!tensors.empty(), "expected a non-empty list of Tensors");
```

Comment:
```suggestion
  TORCH_CHECK_VALUE(!tensors.empty(), "expected a non-empty list of Tensors");
```
These should all be ValueErrors

Reply:
I'll replace all of them in https://github.com/pytorch/pytorch/pull/155460